src/test/java/org/mifos/connector/airtel/AirtelConnectorApplicationTests.java

```diff
@@ -0,0 +1,12 @@
+package org.mifos.connector.airtel;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class AirtelConnectorApplicationTests {
```

Comment:
you can add @openapigenerator here for mainitaining open api spec

Reply:
I could not find any such annotation. Is there some other dependency required other that `org.springdoc:springdoc-openapi-ui`?

Reply:
you can ignore that for now

Reply:
Okay.